.github/scale-config.yml

```
@@ -63,6 +63,12 @@ runner_types:
     is_ephemeral: false
     max_available: 400
     os: linux
+  linux.g4.12xlarge.nvidia.gpu:
+    disk_size: 150
+    instance_type: g4dn.12xlarge
+    is_ephemeral: false
+    max_available: 30
```

```suggestion
    max_available: 10
```

Let's start out with a max of 10 right now and see how that works for us